pkgs/tools/filesystems/gocryptfs/default.nix

```
@@ -18,10 +20,18 @@ buildGoPackage rec {
   src = fetchFromGitHub {
     owner = "rfjakob";
     repo = "gocryptfs";
-    rev = version;
-    sha256 = "0aqbl25g48b4jp6l09k6kic6w3p0q7d9ip2wvrcvh8lhnrbdkhzd";
+    rev = "v${version}";
+    sha256 = "1sr3i73haw07faqpw785cdda2kna8q3a0zhwab1p3i935rvp4qaa";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-unix2syscall_darwin.go-build-failure.patch";
+      url = "https://github.com/rfjakob/gocryptfs/commit/b1468a732fa26550f2a6f8a21cc7bd47b65a8c96.patch";
+      sha256 = "1c9yjnidz47v0kslrip2938cvkwshqin2739561imjcs0d7rijik";
```

```suggestion
      sha256 = "19nw21h07s7w7bncwk6c3mlgz4fpms45g5q4s107h1j95igxz4lh";
```

Hash I get locally. `fetchpatch` normalizes patches so, for example, the hash you'd get from `nix-prefetch-url` won't be correct because the end result is different. I usually TOFU it using a placeholder hash.

---

Resolved with 4e2df9e43f87390961171b2f8c367ed83ac45b3b

Hope to keep that in mind for my future me.